closed/get_j9_source.sh

```diff
@@ -127,8 +127,8 @@ for i in "$@" ; do
 done
 
 # clone OpenJ9 repos
-date '+[%F %T] Get OpenJ9 sources'
-START_TIME=$(date +%s)
+echo "Get OpenJ9 sources"
+START_TIME=$(awk 'BEGIN {srand(); print srand()}')
```

Comment:
I'm surprised `awk` doesn't seem to read stdin, but today I learned that if the only pattern is `BEGIN`, `awk` doesn't read input (unless `getline` is used).

I don't think this output (nor the timing) is needed; I think it should simply be deleted (and the comment on line 129 corrected - it's not just about OpenJ9).